soft4pes/control/lin/__init__.py

```diff
@@ -1,3 +1,8 @@
+"""
+Linear control algorithms for power electronic systems.
```

Comment:
Although I know that we call this "linear" now (and the folders/files), but should we rename it to "conventional" to be somewhat broader?

Reply:
I will make a note, this can be addressed if/when greater structural changes are made